docs/topics/media-pipeline.rst

```diff
@@ -578,4 +577,12 @@ above::
             item['image_paths'] = image_paths
             return item
 
+
+To enable your custom media pipeline component you must add its class to the
```

Comment:
```suggestion
To enable your custom media pipeline component you must add its class import path to the
```

as @Gallaecio suggested in https://github.com/scrapy/scrapy/pull/4252#discussion_r360880296